tests/python_tests/multithreaded_client.py

```diff
@@ -40,7 +40,7 @@
 fdb.options.set_knob("min_trace_severity=5")
 
 if not args.skip_so_files:
-    print "Loading .so files"
+    print("Loading .so files")
```

Comment:
Thanks.  I didn't write this, I just modified it.  Didn't realize the python 3 port was so trivial.  ;-)